src/sentry/static/sentry/app/components/assistant/guideDrawer.jsx

```diff
@@ -0,0 +1,95 @@
+import Reflux from 'reflux';
+import PropTypes from 'prop-types';
+import React from 'react';
+import createReactClass from 'create-react-class';
+import ApiMixin from '../../mixins/apiMixin';
+import GuideStore from '../../stores/guideStore';
+import GuideActions from '../../actions/guideActions';
+
+// GuideDrawer is what slides up when the user clicks on a guide cue.
+const GuideDrawer = createReactClass({
+  displayName: 'GuideDrawer',
+
+  propTypes: {
+    guide: PropTypes.object.isRequired,
+    closeHandler: PropTypes.func.isRequired,
+  },
+
+  mixins: [ApiMixin, Reflux.listenTo(GuideStore, 'onGuideStateChange')],
+
+  getInitialState() {
+    return {
+      step: 0,
+    };
+  },
+
+  onGuideStateChange(data) {
+    this.setState({
+      step: data.currentStep,
+    });
+  },
+
+  nextHandler() {
+    GuideActions.nextStep();
```

Comment:
I would also move this into an action creator, it will help us a bit when we migrate from reflux.